### ISSUE TITLE
Provide way to use multiple wait strategies

### ISSUE THREAD

**Issue by mraerino** [testcontainers/testcontainers-go#37]
Sometimes (when the container does more than one thing) multiple wait-strategies need to be used.

I propose solving this by having a `MultipleStrategy` that might be used like this:

```golang
req := tc.ContainerRequest{
    ExposedPorts: []string{"80"},
    WaitingFor: wait.ForMultiple([]wait.Strategy{
        wait.ForHTTP("/health"),
        wait.ForListeningPort("80/tcp"),
    }),
}
```

**Comment by GaruGaru** [comment]
should be solved by #83

**Comment by gianarb** [comment]
Thanks @GaruGaru
